src/types/ApplicationFormField.ts

```diff
@@ -0,0 +1,76 @@
+import { ajv } from '../ajv';
+import type { JSONSchemaType } from 'ajv';
+
+export interface ApplicationFormField {
+  readonly id: number;
+  applicationFormId: number;
+  canonicalFieldId: number;
```

Comment:
This should be the canonical field *CODE* -- not ID -- since the code is what has meaning to external users.

Reply:
(For future archeologists: we decided not to do this)